### DISCUSSION TITLE
Upgrade odoo 16 -> 17 when avaible

### DISCUSSION THREAD

**Discussion by ckruijntjens** [OCA/OpenUpgrade#4382] [General]
Hi All,

When is it possible to upgrade odoo 16 community edition to odoo 17? I have tried it with the 17 branche but i keep getting errors.

"openupgrade 17 is not a valid upgrade directory"

And one question about how this tool works. i did the following.

install a new vm and installed odoo17. backup the odoo 16 database and tried to import it in the odoo17 machine but this errors out.

**Comment by pedrobaeza** [comment]
No, it's not possible to migrate yet, although anyway your procedure seems invalid.

**Reply by ckruijntjens** [reply]
How should it be done then. can you give me a small discription how to do this when it can be done?

**Reply by pedrobaeza** [reply]
Check the documentation: https://oca.github.io/OpenUpgrade/
